src/electron/views/renderer-initializer.ts

```diff
@@ -292,6 +292,8 @@ getPersistedData(indexedDBInstance, indexedDBDataKeysToFetch).then(
             unifiedResultToIssueFilingDataConverter: null, // we don't support issue filing right now
             windowUtils: null,
             setFocusVisibility,
+            customCongratsMessage:
+                "No failed automated checks were found. Continue investigating your app's accessibility compliance through manual testing",
```

Comment:
is there a missing `.` at the end of the string?

Reply:
Yes. I'l add it to both strings.